firefox-ios/Client/Frontend/Home/PocketFooterView.swift

```diff
@@ -89,6 +87,6 @@ class PocketFooterView: UICollectionReusableView, ReusableCell, ThemeApplicable
     func applyTheme(theme: Theme) {
         let colors = theme.colors
         titleLabel.textColor = wallpaperManager.currentWallpaper.textColor
-        learnMoreLabel.textColor = colors.textAccent
+        learnMoreButton.titleLabel?.textColor = colors.textAccent
```

Comment:
The color doesn't need to be set here as the `LinkButton` already uses the correct colors.

Reply:
I think we need to do learnMoreButton.applyTheme(theme: theme) instead cause without it we get a transparent button. This is also used throughout the codebase for learnMoreButton's